trapping-rainwater.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Write a program that computes how much watera a elevation map is able to trap after raining."
+source: "Trapping Rain Water"
+source_url: "https://leetcode.com/problems/trapping-rain-water"
```

Comment:
leetcodes [TOS][] states:

> # Intellectual Property
>
>The Site and its original content, features and functionality are owned by LeetCode and are protected by international copyright, trademark, patent, trade secret and other intellectual property or proprietary rights laws.

Is this exercise covered by the term “original content”? If so, do we have permission to use the exercises idea, description and graphics?

[TOS]: https://leetcode.com/tos/